apps/happy-blocks/src/support-page/view.scss

```diff
@@ -0,0 +1,151 @@
+@import "@automattic/typography/styles/variables";
+
+.hb-support-page-embed {
+	border-radius: 4px;
+	padding: 29px 30px;
+
+	display: flex;
+	gap: 16px;
+	flex-direction: column;
+
+	background: #fff;
+	border: 1px solid #eee;
+
+	font-family: "SF Pro Text", $sans;
+	font-style: normal;
+	font-weight: 400;
+	font-size: $font-body-small;
+	line-height: 20px;
+
+	a {
+		text-decoration: none !important;
+	}
+
+	&__header {
```

Comment:
Haven't worked in Calypso for a long time but I remember we prefer to not nest selectors (except modifiers and things like `:hover`, etc). Could you move these to be top level ones such as `.hb-support-page-embed__header` etc? More info on https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md.

Reply:
These are not the "component" selectors, but "element" selectors. These elements are parts of the `hb-support-page-embed` component and could not be reused outside. So it conforms to BEM, and Calypso guidelines have it in the first example.

But I do have this issue in `edit.scss`, fixed it.